scripts/hudson/quickstart.sh

```diff
@@ -140,15 +140,19 @@ function configure_wildfly {
   cd $WORKSPACE
   wget --user=guest --password=guest -nv https://ci.wildfly.org/httpAuth/repository/downloadAll/WF_Nightly/.lastSuccessful/artifacts.zip
   unzip -q artifacts.zip
-  export WILDFLY_DIST_ZIP=$(ls wildfly-*Final-SNAPSHOT.zip | head -n 1)
-  [ "x$WILDFLY_DIST_ZIP" = "x" ] && echo "Cannot find WildFly distribution zip file" && return 1
-  unzip -q $WILDFLY_DIST_ZIP
-  export JBOSS_HOME=`pwd`/${WILDFLY_DIST_ZIP%.zip}
+  # the artifacts.zip may be wrapping several zip files: artifacts.zip -> wildfly-latest-SNAPSHOT.zip -> wildfly-###-SNAPSHOT.zip
+  local wildflyLatestZipWrapper=$(ls wildfly-latest-*.zip | head -n 1)
+  unzip -q $wildflyLatestZipWrapper
+  rm -f $wildflyLatestZipWrapper
+  local wildflyDistZip=$(ls wildfly-*-SNAPSHOT.zip | head -n 1)
+  [ "x$wildflyDistZip" = "x" ] && echo "Cannot find WildFly distribution zip file" && return 1
```

Comment:
Did you mean "Cannot find the latest one"?
Also if the zip contains multiple distributions could we use one of the others?

Reply:
I meant any snaphost distribution. I rephrased the error message.
`Also if the zip contains multiple distributions could we use one of the others`, no we use the snapshot one provided by the nightly build. For this CI job we want to use the last nightly one. I don't think we need any choice for this type of CI job.